code_folder/A1_csv_to_json.py

```diff
@@ -0,0 +1,356 @@
+import pandas as pd
+import os
+import json
+from pathlib import Path
+import logging
+
+"""
+How to use:
+all default input csv"s are stored in "/mvs_eland/inputs/elements/default_csv".
+These csvs are not to be changed! All csvs that you need in order to set up 
+your energy system should be stored into "/mvs_eland/inputs/elements/csv". Here
+you can change parameters and add components. The given parameters need to be 
```

Comment:
"Here you can change parameters and add components." 
Where is here? Refer to function or line or keyword (eg. "Hardcoded parameter list")

Reply:
"Here" refers to the directory named in the sentence before. For clarity I replaced it with the following: 

```suggestion
In this directory you can change parameters and add components within the csv 
files. 
```